test/ring_logger/configuration_test.exs

```diff
@@ -0,0 +1,67 @@
+defmodule RingLogger.Configuration.Test do
+  use ExUnit.Case, async: true
+  alias RingLogger.Configuration, as: Subject
+
+  describe "when there is no log level for the module" do
+    setup do
+      {:ok, subject: %{_: :info}}
+    end
+
+    test "when the log level is exceded", %{subject: subject} do
+      assert Subject.meet_level?(subject, TestModule, :warn)
+    end
+
+    test "when the log level is not exceded", %{subject: subject} do
+      refute Subject.meet_level?(subject, TestModule, :debug)
+    end
+  end
+
+  describe "where there is no log level set" do
+    setup do
+      {:ok, subject: %{module_levels: %{}}}
+    end
+
+    test "all log levels are meet", %{subject: subject} do
+      assert Subject.meet_level?(subject, TestModule, :error)
+      assert Subject.meet_level?(subject, TestModule, :warn)
+      assert Subject.meet_level?(subject, TestModule, :info)
+      assert Subject.meet_level?(subject, TestModule, :debug)
+    end
+  end
+
+  describe "where the module is set higher than the log level" do
+    setup do
+      {:ok, subject: %{TestModule => :info, _: :warn}}
+    end
+
+    test "when the log level is meet and the module is not", %{subject: subject} do
+      assert Subject.meet_level?(subject, TestModule, :warn)
+    end
+
+    test "when the log level and the module level are exceded", %{subject: subject} do
+      assert Subject.meet_level?(subject, TestModule, :info)
+    end
+
+    test "when the log level and the module level are not exceded", %{subject: subject} do
+      refute Subject.meet_level?(subject, TestModule, :debug)
+    end
+  end
+
+  describe "where the module is set lower than the log level" do
+    setup do
+      {:ok, subject: %{TeestModule => :warn, _: :info}}
```

Comment:
`TeestModule` -> `TestModule`. I'll have to look harder at this, but I'm surprised this passed the tests...

Reply:
It isn't a real module. The module is lower than the default so it will always run, I think is the issue. I will have to get back to this a little later.

Reply:
fixed